Fred2/EpitopeAssembly/EpitopeAssembly.py

```diff
@@ -339,36 +340,36 @@ def __init__(self, peptides, cl_pred, ep_pred, alleles, threshold, comparator, l
                     fragments[frag] = (start_str, stop_str)
                     fragments[garf] = (stop_str, start_str)
 
-            epi_pred = ep_pred.predict(generate_peptides_from_proteins(fragments.keys(), length), alleles=_alleles)
+            epi_pred = ep_pred.predict(generate_peptides_from_proteins(list(fragments.keys()), length), alleles=_alleles)
```

Comment:
We could remove it it seems, though I am not sure I want to make that call because its not completely the same. I would leave it..
https://stackoverflow.com/questions/16819222/how-to-return-dictionary-keys-as-a-list-in-python

Reply:
In [12]: str(mydict.values())
Out[12]: "dict_values(['apple', 'bee'])"

In [13]: str(list(mydict.values()))
Out[13]: "['apple', 'bee']"

We generate a string of the proteins in generate_peptides_from_proteins() with str() which returns something a bit different, so we should leave the list there

Reply:
It is called once, so I removed the list